packaging/centos-7/build-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alanfranz/fwd-centos-7:latest
```

Comment:
The image I am using is designed to build RPMs for CentOS 7 using fpm within Docker.

I could probably start with a CentOS 7 image, install fpm and Docker, and configure them -- but the odds that I'd end up with something better than this are amazingly low.